README.md

```diff
@@ -4,17 +4,61 @@ We are creating a nice collection of common mistakes when implementing MPC proto
 
 ## Mistakes
 
-### Point-to-Point Channels without Encryption
-
-### Using Multicast Rather than Broadcast
-
-i.e. a broadcast channel cannot be instantiated by simply sending the message to all parties.
-
 ### Commitments are not Bound to the Opening Party
 
+I.e. missing identity information on the sender and session id to ensure that rushing adversaries cannot commitments and opening information from other parties.
+
 ### Zero Knowledge Proofs not Bound to the Protocol Execution
 
+i.e. not embedding the unique context for the protocol _and_ given execution _and_ constructing party (and potentially receiving party).
+
 ### Sequentially Secure Protocol used Concurrently
 
 **Examples:**
 - Blind schnorr signatures and ROS.
+
+### Pitfalls for UC protocols
+Ensure the following:
```

Comment:
if we're describing pitfalls (I think we are), then I guess we should write down mistakes and not best practices.
What I'm getting at: we should either use DOs or DONTs in the whole document consistently.

Reply:
So is this a comment about wording? 
I think we are trying to follow [OWASP](https://owasp.org/www-project-top-ten/) in the sense of mentioning the common pitfalls and best practice. 
To me it seems a bit weird to not give the recommendations when we have them. But I can remove the recommendations if people don't want it here

Reply:
Yeah, this is just about wording.

Above we're listing common mistakes (that one should avoid), here we're listing properties that we want the protocols to have / that one should do. 
Maybe we can do both, e.g.:

> 
> ### Topic 
> Common Pitfall: Missing info in XYZ, allowing reuse.
> Recommendation: Include all the info, tying XYZ to a specific session and making unintended use in other places impossible.